### DISCUSSION TITLE
I'm new to github but i would like to report someone for maleware spreading in code wise.

### DISCUSSION THREAD

**Discussion by CynixWasHere** [community/community#31821] [Other Feature Feedback, Questions, & Ideas]
https://www.virustotal.com/gui/file/1c73d1e40d87c88c67091014ec031d77de049581d4cb6efa4f2cafaceea91f0f/details
https://github.com/Yeethan69/AIO
----------------------------------------
This dude's youtube page is full of bots/half of them are real people because they're saying it's malware, I just want people to be safe instead of their pc getting destroyed by script kiddies.  (I just need a staff to delete their page)

**Comment by jfullstackdev** [comment]
report the user by going to GitHub his/her profile,

**Reply by CynixWasHere** [reply]
ok thanks
